core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -401,7 +401,41 @@ export class TranslationModalComponent {
     return this.translateTextService.getActiveIndex() > 0;
   }
 
-  skipActiveTranslation(): void {
+  getTrimmedTranslationContent(): string {
+    const content = Array.isArray(this.activeWrittenTranslation)
+      ? this.activeWrittenTranslation.join('')
+      : this.activeWrittenTranslation || '';
+    return content.trim();
+  }
+
+  confirmUnsavedChanges(isSaveDisabled: boolean): boolean {
+    if (isSaveDisabled) {
```

Comment:
We try to avoid having verbs at the start of boolean variable names, so perhaps saveButtonIsDisabled is a better name for this.

Why do we check for this though? It seems like we would still want to show the confirmation popup even if the user has submitted partial changes.